ouroboros-network/src/Ouroboros/Cardano/Diffusion/Configuration.hs

```diff
@@ -0,0 +1,50 @@
+
+-- | One stop shop for configuring diffusion layer for upstream clients
+-- This module contains Cardano specific configuration parameters
+
+module Ouroboros.Cardano.Diffusion.Configuration
+  ( DefaultNumBootstrapPeers (..)
+  , NumberOfBigLedgerPeers (..)
+  , defaultNumBootstrapPeers
+  , defaultSyncTargets
+  , defaultNumberOfBigLedgerPeers
+  ) where
+
+import Cardano.Network.Types (NumberOfBigLedgerPeers (..))
+import Ouroboros.Network.Diffusion.Configuration (defaultDeadlineTargets)
```

Comment:
oh, we already have that module - so why not combine both of them?

Reply:
One of them has non-cardano specific values configurations